code/headers/display_adapter.hpp

```diff
@@ -61,13 +87,124 @@ namespace r2d2::display {
          * @param data
          */
         virtual void set_pixels(uint16_t x, uint16_t y, uint16_t width,
-                                uint16_t height, const uint16_t data);
+                                uint16_t height, const uint16_t data) {
+            // set all the pixels to data
+            for (size_t t_y = 0; t_y < width; t_y++) {
+                for (size_t t_x = 0; t_x < width; t_x++) {
+                    // set the pixel with datas
+                    set_pixel(x + t_x, y + t_y, data);
+                }
+            }
+        }
+
+
+        /**
+         * @brief Sets character in a single color
+         * 
+         * @param x x-coordinate of the character (x=0 is the leftmost collumn)
+         * 
+         * @param y y-coordinate of the character (y=0 is the highest row)
+         * 
+         * @param character The un-extended (0-127) ascii value of the character
+         *  
+         * 
+         * @param pixel_color The color of the pixel
```

Comment:
Why all the new lines between the params?

Reply:
What is the problem with this?

Reply:
because this isn't conform the other doxygen in the file. And it looks strange with all the newlines as if you forgot to add something. Furthermore the clang format is configured to remove double newlines and I count this as a double new line. PS this file is not formatted either

Reply:
fixed in c2dfd0d4e42b03c5f9eeb9234d5f7f92fd062ad3